client/src/components/Button/Button.stories.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
-
+import './Button.css'
 import Button from './Button';
 
+function myFunction (){
```

Comment:
When creating functions in a library like this and you should use descriptive names that inform other what exactly this function is doing. You could do something like `showMoreInformation` or `InfoDropDownTrigger`